### PR TITLE
added meta tags, tittle, favicon

### PR DESCRIPTION
**Pull request by manchikantisahith**
changes:
-update the title tag
-addes icon and meta banner
-meta tags includimg og and twitter tag
added fav icon link tag

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
+    <link rel="icon" href="/jovian_favicon.png" type="image/x-icon">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Learn about job openings at Jovian and submit your application now">
+    <meta name="keywords" content="data science, software development, jobs">
+    <meta name="author" content="Jovian">
+    <meta name="robots" content="index, follow">
+    <meta property="og:title" content="Jovian Careers">
+    <meta property="og:description" content="Learn about job openings at Jovian and submit your application now">
+    <meta property="og:image" content="/jovian_meta.png">
```
Comment (MeghanaKolisetty) [inline src/index.html:15]: Make the image some other image

**Review by MeghanaKolisetty — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (manchikantisahith) [reply src/index.html:15]: if you want to see other images please look into google

Comment (MeghanaKolisetty) [reply src/index.html:15]: Gud

**Review by MeghanaKolisetty — APPROVED** [review]

File: src/index.html (hunk 1 above)
Comment (manchikantisahith) [reply src/index.html:15]: good

**Pull request merged**